lib/i18n-spec/matchers/be_a_complete_translation_of_matcher.rb

```diff
@@ -1,14 +1,32 @@
-RSpec::Matchers.define :be_a_complete_translation_of do |default_locale_filepath|
+RSpec::Matchers.define :be_a_complete_translation_of do |default_locale_filepath, check_interpolation_key = false|
```

Comment:
I don't know what the minimum version of Ruby is that is supported by this library, but it'd be great to use a keyword argument here of `check_interpolation_key: false`. I also have a feeling that a default value of `true` would be more useful.

Reply:
I just dropped support for EOL'd rubies, so we can actually apply this suggestion now :+1: